### PR TITLE
Add World Database on Protected Areas layer

### PR DESCRIPTION
**Pull request by mike-mcgann**
## Description

Fixes nasa-gibs/worldview#603.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if appropriate)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Lint and unit tests pass locally with my changes (`npm run test`)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

@nasa-gibs/worldview

### CODE REVIEW

**Comment by mike-mcgann** [comment]
Needs a layer description

File: common/colormaps/wdpa/World_Database_on_Protected_Areas.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ColorMaps xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:noNamespaceSchemaLocation="http://gibs.earthdata.nasa.gov/schemas/ColorMap_v1.3.xsd">
+  <ColorMap title="Protected Areas">
```
Comment (rboller) [inline common/colormaps/wdpa/World_Database_on_Protected_Areas.xml]: This should probably be titled `Protected Area` since it denotes one area at at time.

**Review by rboller — CHANGES_REQUESTED** [review]
Layer looks good but should be supported in image download.

Also, is this layer also available in EPSG:3413?  I could see it being useful for showing areas in higher latitudes.

Otherwise made one small request for the colormap.

**Comment by mike-mcgann** [comment]
Changed areas to area and added to epsg:3413. Not sure how we handle image download these days. I'm assuming we enter in a GIBS ticket?

**Comment by mike-mcgann** [comment]
Ticket GIBS-3129

**Comment by mike-mcgann** [comment]
GIBS ticket got rejected. Now the ticket is WVS-165.

**Comment by ZachTRice** [comment]
Closing this ticket. Please refer to equivalent issues in the [worldview](http://github.com/nasa-gibs/worldview) repository.

**Pull request closed without merging**